### ISSUE TITLE
Accessibility Issues: Admin Dashboard

### ISSUE THREAD

**Issue by aoik109** [lokeshsandhu/Surrey-Food-Bank-Team-c#47]
In the Admin Dashboard, there are 10 failed instances in the automated checks from "Accessibility Insights" involving:

- Discernible text in buttons
- Color contrast issues
- Alternative text in image elements
- Missing labels in form elements

For the Tab stops check, there are 2 failed instances involving:
- Some interactive elements cannot be reached with the "tab" and arrow keys
- There is a non-visible element that the tab after (13) navigates to. This might be the Show only booked timeslots switch.

<img width="1891" height="900" alt="Image" src="https://github.com/user-attachments/assets/f52582a0-9810-4048-a576-f842620b404c" />

Fast Pass Report:
[2026-03-12 Admin Dashboard - FastPass2.html](https://github.com/user-attachments/files/25961453/2026-03-12.Admin.Dashboard.-.FastPass2.html)
